src/store/gateway/plasma.ts

```diff
@@ -334,7 +334,11 @@ export async function plasmaWithdraw(context: ActionContext, funds: Funds) {
       return
     }
     feedback.endTask()
-    feedback.showError(i18n.t("feedback_msg.error.withdraw_failed").toString())
+    if (error.message.includes("limit reached")) {
```

Comment:
Gateway errors have codes (TGXXX), match those, not the description:
- `TG024: total daily withdrawal limit reached`
- `TG025: account daily withdrawal limit reached`